modules/capi-yaml-machine-template-bare-metal.adoc

```diff
@@ -11,16 +11,17 @@ The compute machine set references this template when creating machines.
 
 [source,yaml]
 ----
-apiVersion: infrastructure.cluster.x-k8s.io/v1
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: Metal3MachineTemplate # <1>
 metadata:
-  name: <template_name> # <2>
+  name: <template_name>  # <2>
   namespace: openshift-cluster-api
 spec:
   template:
     spec: # <3>
       customDeploy: install_coreos
-      userData: worker-user-data
+      userData: 
+        name: worker-user-data-managed
```

Comment:
We should add a callout for `worker-user-data-managed`. 
`It is an ignition configuratin generated and managed by Machine API operator for baremetal IPI clusters. This secret should be applied in opensift-cluster-api namespace from the secret generated at cluster installation in opensihft-machine-api namespace. To do this use the command`
```
$ oc get secret worker-user-data-managed -n openshift-machine-api -o yaml | sed 's/namespace: .*/namespace: openshift-cluster-api/' | oc apply -f -
```
@honza please review

Reply:
@johnwilkins Please address QE's comment. Callouts are currently supported.